cli/src/core/parse/profile.go

```diff
@@ -0,0 +1,48 @@
+package parse
+
+import (
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+func GetPackageSpecFromProfile(cmd *cobra.Command, config core.Config, packageSpec core.PackageSpec) (*core.PackageSpec, error) {
+	profile := core.Profile{}
+	profileName, err := cmd.Flags().GetString("profile")
+	if err != nil {
+		return nil, errors.Wrap(err, "")
+	}
+
+	for _, p := range config.Profiles {
+		if p.Name == profileName {
+			profile = p
+			break
+		}
+	}
+
+	if !cmd.Flags().Changed("dev") && profile.Dev {
+		packageSpec.IsDev = profile.Dev
+	}
+
+	if !cmd.Flags().Changed("only") && profile.Only {
+		packageSpec.IsOnly = profile.Only
+	}
```

Comment:
The  `isOnly` and `isDev` properties should also be updated when they are specified on the command line

Reply:
This function will update according to profile only, the properties was already updated according to the command.

We agreed  that the command line has more priority so here he is just updating the flags in case we didn't specify the flags in the command